### PR TITLE
added modes

### PR DESCRIPTION
**Pull request by DTau9**
Hello! I added original confetti modes for your react-canvas-confetti library.

### CODE REVIEW

**Comment by DTau9** [comment]
@ulitcos  ?

File: package.json [hunk 1]
```diff
@@ -26,9 +26,9 @@
   "devDependencies": {
     "@babel/core": "7.12.10",
     "@storybook/addon-actions": "6.1.11",
-    "@storybook/addon-essentials": "6.1.11",
+    "@storybook/addon-essentials": "^6.1.21",
     "@storybook/addon-links": "6.1.11",
-    "@storybook/react": "6.1.11",
+    "@storybook/react": "^6.1.21",
```
Comment (ulitcos) [inline package.json:31]: I'm planing save dependencies in static

**Review by ulitcos — CHANGES_REQUESTED** [review]

File: src/stories/fireworks/Fireworks.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Fireworks from './Fireworks';
+
+export default {
+  title: 'Fireworks mode',
```
Comment (ulitcos) [inline src/stories/fireworks/Fireworks.stories.tsx:5]: Fireworks example?

File: src/stories/fireworks/Fireworks.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  private intervalId: NodeJS.Timeout | null = null;
+
+  constructor(props: IFireworks) {
+    super(props);
+    this.isAnimationEnabled = false;
+    this.nextTickAnimation = this.nextTickAnimation.bind(this);
+  }
+
+  getAnimationSettings(originXA: number, originXB: number) {
+    return Object.assign({},
```
Comment (ulitcos) [inline src/stories/fireworks/Fireworks.tsx:24]: Why are assign here?

File: src/stories/fireworks/Fireworks.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  render() {
+    return (
+      <>
+        <div className="canvas-confetti-modes__control">
+          <button role="option" onClick={this.handlerClickStart}>Start</button>
+          <button role="option" onClick={this.handlerClickPause}>Pause</button>
+          <button role="option" onClick={this.handlerClickStop}>Stop</button>
+        </div>
+        <ReactCanvasConfetti
+          refConfetti={(instance) => this.animationInstance = instance}
```
Comment (ulitcos) [inline src/stories/fireworks/Fireworks.tsx:87]: Why create every render?

File: src/stories/realisticConfetti/RealisticConfetti.tsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+      spread: 120,
+      startVelocity: 45,
+    });
+  }
+
+  render() {
+    return (
+      <>
+        <div className="canvas-confetti-modes__control">
+          <button onClick={() => this.fire()}>Fire</button>
```
Comment (ulitcos) [inline src/stories/realisticConfetti/RealisticConfetti.tsx:55]: Create new every render

File: src/stories/realisticConfetti/RealisticConfetti.tsx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+export default class RealisticConfetti extends Component {
+  private animationInstance: CreateTypes | null = null;
+
+  constructor(props: IRealisticConfetti) {
+    super(props);
+    this.fire = this.fire.bind(this);
+  }
+
+  makeShot(particleRatio: number, opts: object) {
+    this.animationInstance && this.animationInstance(Object.assign({}, { origin: { y: 0.7 } }, opts, {
```
Comment (ulitcos) [inline src/stories/realisticConfetti/RealisticConfetti.tsx:17]: Object.assign?

**Review by ulitcos — CHANGES_REQUESTED** [review]

**Pull request merged**